projects/swimlane/ngx-ui/src/lib/components/toolbar/toolbar.component.ts

```diff
@@ -83,7 +48,8 @@ export class ToolbarComponent {
   }
   constructor(private renderer: Renderer2, private elRef: ElementRef) {}
 
-  onMenuClicked(item, $event) {
+  onMenuClicked(item: ToolbarMenuItem, $event: any) {
+    /* istanbul ignore else */
```

Comment:
we should either test this path and remove the ignore, or remove the if statement if `item.click` is not nullable

Reply:
I added a test thatll test if the click function was not defined for a menu item